mautic/mautic
# TPROD-272: Dropped PluginConfiguration.php

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
Fix  https://mautic.atlassian.net/browse/TPROD-272
Blocked for Symfony 5. Seems like we do not use it anywhere



1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check If installation works like before and plugins reload works too


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11032"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
